linux/Platform.c

```diff
@@ -614,6 +614,7 @@ bool Platform_getDiskIO(DiskIOData* data) {
    char lastTopDisk[32] = { '\0' };
 
    unsigned long long int read_sum = 0, write_sum = 0, timeSpend_sum = 0;
+   unsigned long long int numDisks = 0;
```

Comment:
Possible cleanup: Be consistent with later usage of this variable and use `uint64_t` here too …

Also, for consistency with other platforms, a blank line after these two lines would be good.